src/ts/Ride/FormWrap.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+export default class FormWrap extends React.Component<any, any> {
+  requestId: number;
+  state = { form: '' }
+
+  componentDidMount() {
+    this.handleScroll();
+  }
+
+  componentWillUnmount() {
+    window.cancelAnimationFrame(this.requestId);
```

Comment:
Yep, for tests I added in `test-setup.js` `requestAnimationFrame` that returns 2 (`requestId`). On server-side rendering methods like `componentDidMount` or `componentWillUnmout` aren't invoked.